src/olympia/reviewers/views.py

```diff
@@ -1037,7 +1038,7 @@ def abuse_reports(request, addon):
         Q(addon=addon) | Q(user__in=developers)
     ).select_related('user').prefetch_related(
         # See review(): we only need the add-on objects and their translations.
-        Prefetch('addon', queryset=Addon.objects.all().only_translations()),
+        Prefetch('addon', queryset=Addon.unfiltered.all().only_translations()),
```

Comment:
We need a test that proves this change works.  There's no prescriptive way to do this, but I'd add another test in `TestAbuseReportsView` in `reviewers/tests/test_views.py` that deletes `self.addon` and tries to do `test_abuse_reports` again (or something like it)